dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.mpHealthCompatible-0.0.feature

```diff
@@ -0,0 +1,8 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.mpHealthCompatible-0.0
+WLP-DisableAllFeatures-OnConflict: false
```

Comment:
`mpHealth-1.0` and `mpHealth-2.x`  are configured so that the `disableAllFeatures` is false. Setting this one to false to avoid the visibility failures.